gigadb/app/tools/transfer-files/configure

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
question: We do not need this configure script because we do not need it to create a rclone.conf file?

Reply:
**answer:** We still need it to generate `.env` and `.secrets` to execute the `transfer.sh` locally.